apps/elasticsearch/templates/services/app/sts.yml.j2

```diff
@@ -42,6 +42,44 @@ spec:
                       values:
                         - "elasticsearch-node"
                 topologyKey: kubernetes.io/hostname
+{% if elasticsearch_security_enabled %}
+      initContainers:
+        - image: {{ elasticsearch_image_name }}:{{ elasticsearch_image_tag }}
+          name: init-generate-tls-certs
+          command:
+            - "bash"
+            - "-c"
+            - mkdir -p "{{ elasticsearch_ssl_certificates_path }}" &&
+              elasticsearch-certutil cert
+                --name "${HOSTNAME}"
+                --dns "${HOSTNAME},elasticsearch"
```

Comment:
`elasticsearch` is the name of the service that we will request. Should it be configurable?